pkg/login/auth.go

```diff
@@ -25,14 +27,34 @@ var (
 
 var loginLogger = log.New("login")
 
-var AuthenticateUserFunc = AuthenticateUser
+type Authenticator interface {
+	AuthenticateUser(context.Context, *models.LoginUserQuery) error
+}
+
+var AuthenticateUserFunc = func(context.Context, *models.LoginUserQuery) error {
+	panic("AuthenticateUserFunc is not implemented")
+}
 
 func Init() {
-	bus.AddHandler("auth", AuthenticateUser)
+}
```

Comment:
Remove?

Reply:
Yes, the Init is empty. If we want to keep it it should be called inside the Provide function like we are doing in other services.